src/main/scala/mesosphere/marathon/state/Volume.scala

```diff
@@ -364,7 +361,6 @@ object ExternalVolumeInfo {
 
   implicit val validExternalVolumeInfo = validator[ExternalVolumeInfo] { info =>
     info.size.each should be > 0L
-    info.name should matchRegex(LabelRegex)
```

Comment:
instead of removing validation out-right, can we limit it to URI characters perhaps? Hesitant to allow surrounding whitespace, for example.